### PR TITLE
Update FRENCH translation + fix style.

### PR DESCRIPTION
**Pull request by Galixte**
Hi @AlexSheer,

could you merge this, please?

### CODE REVIEW

File: styles/prosilver/template/kb_cat_body.html [hunk 1]
```diff
@@ -96,9 +96,9 @@
...
-						<!-- IF not art_row.S_APPROVED -->	<li><a href="{art_row.U_ARTICLE}"	title="{L_NEED_APPROOVE}"	class="button button-icon-only"><i class="icon fa-check fa-fw" aria-hidden="true"></i><span class="sr-only">{L_NEED_APPROOVE}</span></a></li><!-- ENDIF -->
 						<!-- IF art_row.S_CAN_DELETE -->	<li><a href="{art_row.U_DELETE}"	title="{L_DELETE}"			class="button button-icon-only"><i class="icon fa-times fa-fw" aria-hidden="true" style="color: #AA0000;"></i></i><span class="sr-only">{L_DELETE}</span></a></li><!-- ENDIF -->
 						<!-- IF art_row.S_CAN_EDIT -->		<li><a href="{art_row.U_EDIT_ART}"	title="{L_EDIT}"			class="button button-icon-only"><i class="icon fa-pencil fa-fw" aria-hidden="true"></i><span class="sr-only">{L_EDIT}</span></a></li><!-- ENDIF -->
+						<!-- IF not art_row.S_APPROVED -->	<li><a href="{art_row.U_ARTICLE}"	title="{L_NEED_APPROOVE}"	class="button button-icon-only"><i class="icon fa-check fa-fw" aria-hidden="true"></i><span class="sr-only">{L_NEED_APPROOVE}</span></a></li><!-- ENDIF -->
```
Comment (Galixte) [inline styles/prosilver/template/kb_cat_body.html:101]: @AlexSheer, with this change, all MOVE, DELETE and EDIT button are aligned vertically, and if there are some APPROVE buttons displayed these are displayed at the end of buttons list and aligned also vertically.

**Comment by AlexSheer** [comment]
I'm renamed some files from old stupid error

> Conflicting files
> language/fr/info_acp_knowlegebase.php
> language/fr/knowlegebase_lng.php
So, please recheck it

**Comment by Galixte** [comment]
ok i’ll re-submit this PR.

**Comment by Galixte** [comment]
https://github.com/AlexSheer/phpbb3.2-Knowledge-Base/pull/24

**Comment by Galixte** [comment]
@AlexSheer i see your renaming changes but there is an other one, here: https://github.com/AlexSheer/phpbb3.2-Knowledge-Base/blob/master/migrations/version_1_0_0.php#L311

```Sheer/knowlegebase\``` should be ```Sheer/knowledgebase\```, no?

**Comment by AlexSheer** [comment]
No. It's for converting from old version for 3.1.x.

**Comment by Galixte** [comment]
ok, thanks.

**Pull request closed without merging**